packages/@ourworldindata/utils/src/regions.ts

```diff
@@ -0,0 +1,70 @@
+import entities from "./regions.json"
+
+export interface Country {
+    code: string
+    shortCode?: string
+    name: string
+    shortName?: string
+    slug: string
+    regionType?: "country" | "other"
+    isMappable?: boolean
+    isHistorical?: boolean
+    isUnlisted?: boolean
+    variantNames?: string[]
+}
+
+export interface Aggregate {
+    name: string
+    regionType: "aggregate"
+    code: string
+    members: string[]
+}
+
+export interface Continent {
+    name:
+        | "Africa"
+        | "Asia"
+        | "Europe"
+        | "North America"
+        | "Oceania"
+        | "South America"
+    regionType: "continent"
+    code: string
+    members: string[]
+}
+
+export type Region = Country | Aggregate | Continent
+
+export const regions: Region[] = entities as Region[]
```

Comment:
Is `Region`/`regions` actually the right name for this? I considered `Entity` but there are a lot of non-geographical entities in various data sets as well so figured there should be *something* suggesting place/state-ness…